azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/b2c/AADB2CFilterSignUpOrInHandler.java

```diff
@@ -25,42 +31,94 @@
 public class AADB2CFilterSignUpOrInHandler extends AbstractAADB2CFilterScenarioHandler
         implements AADB2CFilterScenarioHandler {
 
+    private final RedirectStrategy redirectStrategy = new DefaultRedirectStrategy();
+
     /**
      * Mapping configuration URL to ${@link AADB2CJWTProcessor}.
      */
-    private final ConcurrentMap<String, AADB2CJWTProcessor> urlToJwtParser = new ConcurrentHashMap<>();
+    private static final ConcurrentMap<String, AADB2CJWTProcessor> URL_TO_JWT_PARSER = new ConcurrentHashMap<>();
```

Comment:
This is not a constant. Original naming is fine.

Reply:
make sense, will revert this.